sites/friday/src/get-started/nextjs/_index.md

```diff
@@ -1,4 +1,518 @@
+
+```bash {location="Terminal"}
+echo "/node_modules" >> .gitignore
+echo "/.next" >> .gitignore
+git add .gitignore && git commit -m "adding node_modules and .next folder in .gitignore file"
+```
+
+## Add a Hello World route
+
+Please create your first Next.js page.
+To do so, create a new ``pages/index.tsx`` file at the root of your project. It will contain a basic Hello world script:
+```javascript {location="pages/index.tsx"}
+export default function Page() {
+  return <h1>Hello world, Next.js!</h1>
+}
+```
+
+Then please commit your file using the following commands:
+```bash {location="Terminal"}
+git add pages/index.tsx
+git commit -m "adding pages/index.tsx first page"
+```
+
+## Create a new project
+
+The next step is to create a project on {{% vendor/name %}}.
+To do this, you can either use the {{% vendor/name %}} CLI or the {{% vendor/name %}} Console.
+
+{{< codetabs >}}
++++
+title=Using the CLI
++++
+To create a new project with the {{% vendor/name %}} CLI, use the following command and follow the prompt:
+```bash {location="Terminal"}
+{{% vendor/cli %}} project:create
+```
+
+{{< note >}}
+When creating a new project using the {{% vendor/name %}} CLI command `project:create`, you will be asked if you want to set the local remote to your new project. Enter **Yes (y)**.
+Your local source code will be automatically linked to your newly created {{% vendor/name %}} project by creating a `.{{% vendor/cli %}}/local/project.yaml` file that will contain the corresponding `<projectId>` and set a Git remote to `{{% vendor/cli %}}`.
+
+```bash {location="Terminal"}
+git remote
+{{% vendor/cli %}}
+```
+
+If not, please refer to [Set project remote](#set-project-remote) section.
+{{< /note >}}
+
+<--->
++++
+title=Using the Console
++++
+
+[Create a new project from scratch]({{% vendor/urlraw "console" %}}/projects/create-project/).
+
+If you do not already have an organization created to put the project, you'll first be instructed to create one.
+
+Once you have done so, select that organization from the dropdown, and select **Create from scratch**.
+
+In the form, fill in details like the project name and [region](/development/regions.md).
+You'll be able to define resources for the project after your first push.
+
+After creating a project with the console, you need to let the {{% vendor/name %}} CLI know which linked project you want to deploy to.
+
+To do so, use the {{% vendor/name %}} CLI to set remote project:
+```bash {location="Terminal"}
+{{% vendor/cli %}} project:set-remote <projectId>
+```
+
+This command will add a new remote called `{{% vendor/cli %}}` to your local Git repo as you can see below:
+```bash {location="Terminal"}
+git remote
+origin
+{{% vendor/cli %}}
+```
+
+It will also create a new `.{{% vendor/cli %}}/local/project.yaml` file that will contain the given `<projectId>`, to store this info for the {{% vendor/name %}} CLI interaction.
```

Comment:
```suggestion
It will also create a new `.{{% vendor/cli %}}/local/project.yaml` file that will contain the given `<projectId>`, to store this information for the {{% vendor/name %}} CLI interaction.
```